.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+
+## Related Issue(s), If Filed
+<!--- What is the related issue you are trying to fix? -->
+
+## Testing
+<!--- Please describe in detail how you tested your changes -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Checklist
+<!--- Go over all the following points, and put an `x` in all the boxes that apply -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the **README** document
+<!-- - [ ] I have read the **CONTRIBUTING** document -->
+- [ ] Local run of `gulp package` succeeds
+- [ ] My code follows the code style of this project
+<!-- 
+- [ ] I have added tests to cover my changes
+- [ ] All new and existing tests passed
+-->
+- [ ] A short description of the change has been added to the **CHANGELOG** (CHANGELOG.md)
+
+## License
+
+By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
```

Comment:
This should look more consistent with the other toolkit repos. Consider:

```
By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.
```

Reply:
This is the message the AWS SDK's use, and was in the old PR template for this product.

Reply:
I take back my comment. I like it in its current form.